server/src/main/java/org/opensearch/indices/ShardLimitValidator.java

```diff
@@ -64,12 +64,23 @@ public class ShardLimitValidator {
         Setting.Property.Dynamic,
         Setting.Property.NodeScope
     );
+
+    public static final Setting<Boolean> SETTING_CLUSTER_IGNORE_HIDDEN_INDEXES = Setting.boolSetting(
+        "cluster.ignore_hidden_indexes",
+        true,
+        Setting.Property.Dynamic,
+        Setting.Property.NodeScope
```

Comment:
I am not super familiar with `NodeScope`. Confirm that this is a cluster-wide setting and that I cannot have it `true` on one node and `false on another?

Reply:
The Node scope setting is per node scope only . However once the active ClusterManagerNode is up the default setting for that particular node is only considered. .

So if one has true and other has false the setting on active ClusterManagerNode is taken. Added a UT for this as well in the new commit.